tools/wltests/build

```diff
@@ -287,41 +288,40 @@ fi
 
 if [ ${BUILD_IMAGE} -eq $YES ]; then
 	c_info "Making [Image]..."
-	(set -x; cd "${KERNEL_SRC}"; make -j${NCPUS} "$KERNEL_IMAGE" >${OUT})
+	(set -x; cd "${KERNEL_SRC}"; make ${BUILD_FLAGS} -j${NCPUS} "${KERNEL_IMAGE}" >${OUT})
 fi
 
 if [ ${BUILD_DTBS} -eq $YES ]; then
 	c_info "Making [dtbs]..."
-	(set -x; cd "${KERNEL_SRC}"; make -j${NCPUS} dtbs >${OUT})
+	(set -x; cd "${KERNEL_SRC}"; make ${BUILD_FLAGS} -j${NCPUS} dtbs >${OUT})
 fi
 
 if [ ${BUILD_DTBO_IMAGE} -eq $YES ]; then
 	c_info "Making [dtbo.img]..."
-	(set -x; cd "${KERNEL_SRC}"; make -j${NCPUS} dtbo.img >${OUT})
+	(set -x; cd "${KERNEL_SRC}"; make ${BUILD_FLAGS} -j${NCPUS} dtbo.img >${OUT})
 fi
 
 if [ ${BUILD_MODULES} -eq $YES ]; then
 	c_info "Making [modules]..."
-	(set -x; cd "${KERNEL_SRC}";  make -j${NCPUS} modules >${OUT})
+	(set -x; cd "${KERNEL_SRC}";  make ${BUILD_FLAGS} -j${NCPUS} modules >${OUT})
 fi
 
 if [ ${BUILD_MODULES_INSTALL} -eq $YES ]; then
 	c_info "Making [modules_install] to [${INSTALL_MOD_PATH}] ..."
-	(set -x; cd "${KERNEL_SRC}";  make -j${NCPUS} modules_install >${OUT})
+	(set -x; cd "${KERNEL_SRC}";  make ${BUILD_FLAGS} -j${NCPUS} modules_install >${OUT})
 fi
 
 if [ ${MAKE_MENUCONFIG} -eq $YES ]; then
 	c_info "Making [menuconfig]..."
-	(set -x; cd "${KERNEL_SRC}"; make menuconfig)
+	(set -x; cd "${KERNEL_SRC}"; make ${BUILD_FLAGS} menuconfig)
 fi
 
 if [ ${MAKE_SAVEDEFCONFIG} -eq $YES ]; then
 	c_info "Making [savedefconfig]..."
-	(set -x; cd "${KERNEL_SRC}"; make savedefconfig)
+	(set -x; cd "${KERNEL_SRC}"; make ${BUILD_FLAGS} savedefconfig)
```

Comment:
Dumb question: is this needed for defconfig-related stuff ?

Reply:
I would say no, but the build guide uses the same flags for building the config & for building the kernel image, and seeing how everything breaks when I try not to follow the guide I applied this for the defconfig as well...